packages/components/src/styling/src/styled-system/useStyledSystem.ts

```diff
@@ -282,23 +284,28 @@ const BorderRadiusScale = [
     2,
     3,
     4,
```

Comment:
on devrait enlever de 1 a 4 surement

Reply:
1, 2, 3, 4 are the global tokens I would keep them

Reply:
I feel like sm/md/lg are the global tokens. 

An alias would be something like border-radius-cards, border-radius-sections, border-radius-input, where their semantic value would represent a moment to use them. 

the way it is, sm/md/lg brings no value over 1/2/3/4

Reply:
The way I see it is that xs, md etc can refer to our scale, right now we indeed have four but that would allow us to have more and lg could eventually refer to br-6 or 8